css/navbar.css

```diff
@@ -102,6 +107,46 @@ nav li a:hover {
   margin-right: 10px;
 }
 
+.dropdown {
+  width: 180px;
+  background-color: var(--dropdown-background);
+  border: 2px solid var(--dropdown-border);
+  border-radius: 0.5rem;
+  color: var(--dropdown-text);
+  position: absolute;
+  top: 55px;
+  right: 16px;
+  z-index: 100;
+}
+
+.dropdown-list {
+  padding: 0;
+}
+
+.dropdown-link {
+  color: var(--dropdown-text);
+}
+
+.line {
+  border: none;
```

Comment:
`border: none` is not required

Reply:
then we would get border everywhere as I am using a hr which has a class of line

Reply:
it is not a div its a hr so by default it has a border property so I tried overwriting it.